docs/preview/features/writing-different-telemetry-types.md

```diff
@@ -373,6 +373,16 @@ logger.LogMetric("Invoice Received", 133.37, telemetryContext);
 
 Requests allow you to keep track of the HTTP requests that are performed against your API and what the response was that was sent out.
 
+**Installation**
+
+If you want to track the `HttpRequest` and `HttpResponse` of an ASP.NET Core project, you'll have to install an additional package to include these ASP.NET Core dependencies:
+
+```shell
+PM > Install-Package Arcus.Observability.Telemetry.AspNetCore
+```
+
+**Example**
+
```

Comment:
It would be nice to extend the documentation regarding application insights with a more extended code-sample.  Right now, some code is shown on how to log to AppInsights via Serilog, but it is not shown where you could put that code.